website/docs/r/aaep_to_domain.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
+subcategory: "Access Policies"
+layout: "aci"
+page_title: "ACI: aci_aaep_to_domain"
+sidebar_current: "docs-aci-resource-aaep-to-domain"
+description: |-
+  Manages ACI Attachable Access Entity Profile - VMM, Physical or External domain interfaces.
```

Comment:
Manages the ACI Attachable Access Entity Profile (AAEP) to domain (VMM, Physical or External domain) relationship